### ISSUE TITLE
LoRA finetune bad performance?

### ISSUE THREAD

**Issue by HomerIsAFool** [NVIDIA/Isaac-GR00T#367]
Has anyone finetune the model with LoRA and tested the model performance?

I finetune Gr00tN1.5 with LoRA (lora_rank == 2 , lora_alpha ==2), The MSE Loss is around 0.04, which looks fine.
(i only have 2 A6000 GPU)
The self-collected dataset contains around 80 trajectories.

But when i deploy the model on the real robot(Lerobot So101 Dual Arm), the grasping result is really bad.
the robotic arm showed attempt to grasp, but it couldn’t actually grab the object, and there was also severe shaking

i’m not sure whether the issue is due to underfitting caused by using LoRA, or a problem in my deployment code.

Has anyone deployed the model finetuned with Lora and executing the task successfully?

Any answer would be grateful.

Thanks!

**Comment by xwjiang2010** [comment]
Can you show open loop eval result?
In general, LoRA is not as good as full finetune.

**Comment by HomerIsAFool** [comment]
Thank you for your quick reply!
The eval result is as below, which seems not that bad , but the deployment performance is rather poor
<img width="2450" height="1263" alt="Image" src="https://github.com/user-attachments/assets/0c742f46-a674-40eb-bd9c-9574a8a26b5b" />

**Comment by xwjiang2010** [comment]
The openloop eval doesn't look good. It could be that LoRA doesn't offer enough capacity to fit the data or the training steps should be larger.
